servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -141,10 +152,13 @@ public RoundRobinLoadBalancer(final Publisher<? extends ServiceDiscovererEvent<R
      * set to {@code false} should be eagerly closed. When {@code false}, the expired addresses will be used
      * for sending requests, but new connections will not be requested, allowing the server to drive
      * the connection closure and shifting traffic to other addresses.
+     * @param healthCheckConfig configuration for the health checking mechanism, which monitors hosts that
```

Comment:
`healthCheckConfig` is nullable but it is not explained what happens when it is null or why you might not provide a health check config.

Reply:
Thanks for noticing, I added a note about it.

Reply:
What is the result of disabling it? Will the hosts never become eligible?

Reply:
I decided to point to the factory, which is a public API and contains a more thorough explanation.